src/index.html

```diff
@@ -1,13 +1,285 @@
+            </h3>
+
+            <p class="about__paragraph">
+              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+
+            <p class="about__paragraph">
+              The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+
+            <p class="about__paragraph about__paragraph--important about__paragraph--last">
+              We will never be late!
+              <br>
+              Guaranteed secure transaction - 100%
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="page__section gallery" id="gallery">
+      <div class="container">
+        <h2 class="section-title">Gallery</h2>
+
+        <div class="place">
+          <div class="place__photos">
+            <img class="place__photo place__photo--1" src="./images/photos/1.jpg" alt="Place 1">
+            <img class="place__photo place__photo--2" src="./images/photos/2.jpg" alt="Place 2">
+          </div>
+
+          <p class="place__address">
+            <a href="#place1">Marina Palms</a>
+            / North Miami Beach, FL 33162
+          </p>
+        </div>
+
+        <div class="place">
+          <div class="place__photos">
+            <img class="place__photo place__photo--3" src="./images/photos/3.jpg" alt="Place 3">
+            <img class="place__photo place__photo--4" src="./images/photos/4.jpg" alt="Place 4">
+          </div>
+
+          <p class="place__address">
+            <a href="#place1">Marina Palms</a>
+            / North Miami Beach, FL 33139
+          </p>
+        </div>
+
+        <div class="place">
+          <div class="place__photos">
+            <img class="place__photo place__photo--5" src="./images/photos/5.jpg" alt="Place 6">
+            <img class="place__photo place__photo--6" src="./images/photos/6.jpg" alt="Place 5">
+          </div>
+
+          <p class="place__address">
+            <a href="#place1">Marina Palms</a>
+            / North Miami Beach, FL 33140
+          </p>
+        </div>
+
+        <div class="gallery__button-container">
+          <a
+            class="gallery__button button button--secondary"
+            href="#view-all"
+          >
+            View all
+          </a>
+        </div>
+      </div>
+    </section>
+
+    <section class="page__section testimonials" id="testimonials">
+      <div class="container">
+        <h2 class="section-title">Testimonials</h2>
+
+        <div class="testimonials__content">
+          <div class="testimonials__item-1 testimonial">
```

Comment:
better to use just regular class name `testimonials__item` instead of adding suffixes like `item-1, item-2`

Usually, you can be given a request to add one more item or to swap item-1 and item-2

Reply:
agree